qutebrowser/utils/log.py

```diff
@@ -131,7 +129,6 @@ def vdebug(self: logging.Logger,
 keyboard = logging.getLogger('keyboard')
 downloads = logging.getLogger('downloads')
 js = logging.getLogger('js')  # Javascript console messages
-qt = logging.getLogger('qt')  # Warnings produced by Qt
```

Comment:
@The-Compiler, I wonder if we should also move the qt logger's name ([part of `LOGGER_NAMES` below](https://github.com/qutebrowser/qutebrowser/pull/7772/files#diff-dfe7b37bd6ae55b3c36fd5155ce2a378d2351f8e4baba5badfad88c3e0924d0cL148-L157)) to `qutebrowser.util.qtlog`. It feels a bit silly just for a single name, but right now the definitions of all loggers and the definition of `LOGGER_NAMES` is kind of ripped apart, which I am not so happy about.

Something along the lines of:
```python
# qutebrowser/util/qtlog.py
LOGGER_NAMES = ['qt']

# qutebrowser/util/log.py
LOGGER_NAMES = [
    'statusbar', 'completion', 'init', 'url',
    'destroy', 'modes', 'webview', 'misc',
    'mouse', 'procs', 'hints', 'keyboard',
    'commands', 'signals', 'downloads',
    'js', 'qt', 'ipc', 'shlexer',
    'save', 'message', 'config', 'sessions',
    'webelem', 'prompt', 'network', 'sql',
    'greasemonkey', 'extensions',
    *qtlog.LOGGER_NAMES,
]
```
What do you think?

Reply:
I'd probably keep both this line and the corresponding `LOGGER_NAMES` entry in here. While it is Qt specific, it still belongs tightly to the other logger definitions which are in here - and as you mention, ripping them apart is kind of awkward.

Reply:
Right now this can only be solved by duplicating `qt = logging.getLogger('qt')` to `qutebrowser.util.qtlog`. Otherwise we get a circular import by having to import `qt` in `qutebrowser.util.qtlog`:
```python
# qutebrowser/util/qtlog.py

from qutebrowser.util.log import qt  # circular import
```

We should be able to address this once we move `qtlog.init()` out of `qutebrowser.util.log.init_log()`.

Reply:
Just fixed this in 8d14d5cef61f6947b7929194ebb23d8b260855bf.